packages/Input/src/Input.js

```diff
@@ -150,21 +139,18 @@ const propTypes = {
   /** If true it makes the input read only. */
   isReadOnly: PropTypes.bool,
 
-  /**
-   * Callback to be executed when the input value is changed. Receives the
-   * onChange event as an argument, except when the clear button is clicked,
-   * then the argument is null. Needed when value prop is provided (component
-   * is controlled).
-   */
+  /** Callback to be executed when the input value is changed. Receives the onChange event as an argument, except when the clear button is clicked, then the argument is null. Required when value prop is provided (component is controlled). */
   onChange: PropTypes.func,
 
+  /** Ref for the input DOM element. */
+  refInput: RefOf(PropTypes.instanceOf(Element)),
```

Comment:
I stopped using `PropTypes.instanceOf(Element)` as it throws errors in server side starling for example. An Alternative is just to use `PropTypes.node`

reference: https://github.com/acl-services/paprika/pull/1030